modules/gridmap/grid_map.cpp

```diff
@@ -755,6 +765,7 @@ void GridMap::_reset_physic_bodies_collision_filters() {
 	for (const KeyValue<OctantKey, Octant *> &E : octant_map) {
 		PhysicsServer3D::get_singleton()->body_set_collision_layer(E.value->static_body, collision_layer);
 		PhysicsServer3D::get_singleton()->body_set_collision_mask(E.value->static_body, collision_mask);
+		PhysicsServer3D::get_singleton()->body_set_collision_priority(E.value->static_body, collision_priority);
```

Comment:
I'm fine with adding it here, but since the (internal) method name `_reset_physic_bodies_collision_filters` was already using some weird/archaic spelling, and now the name doesn't fully describe it anymore, maybe we can rename it to, say, `_update_physics_bodies_collision_properties` (since it's also not exactly a reset).